src/notifications/types.ts

```diff
@@ -0,0 +1,131 @@
+import { ICourierPaging } from "../types";
+
+export interface ICourierNotificationListResponse {
+  paging: ICourierPaging;
+  results: Array<{
+    id: string;
+    title: string;
+  }>;
+}
+
+export interface ICourierNotificationListParams {
+  cursor?: string;
+}
+
+export type BlockType =
+  | "action"
+  | "divider"
+  | "image"
+  | "jsonnet"
+  | "list"
+  | "markdown"
+  | "quote"
+  | "template"
+  | "text";
+
+export interface ICourierNotificationBlock {
+  alias?: string;
+  context?: string;
+  id: string;
+  type: BlockType;
+  content?: string | { parent?: string; children?: string };
+  locales?: {
+    [locale: string]:
+      | string
+      | {
+          parent?: string;
+          children?: string;
+        };
+  };
+}
+
+export interface ICourierNotificationChannel {
+  id: string;
+  email?: {
+    subject?: string;
```

Comment:
I would appreciate as an end-user if `email`, `push` etc have dedicated interfaces that courier exports so that I can integrate that in my application and can get intelligence.